src/components/App.tsx

```diff
@@ -15,17 +17,25 @@ import { EvaluatedPathway } from 'pathways-model';
 import useGetPathwaysService from './PathwaysService/PathwaysService';
 import FHIR from 'fhirclient';
 import demoRecords from 'fixtures/MaureenMcodeDemoPatientRecords.json';
-
+import { MockedFHIRClient } from 'utils/MockedFHIRClient';
+import { getHumanName } from 'utils/fhirUtils';
 interface AppProps {
   demo: boolean;
 }
 
 const App: FC<AppProps> = ({ demo }) => {
-  const [patientRecords, setPatientRecords] = useState<Array<fhir.DomainResource>>([]);
+  const [patientRecords, _setPatientRecords] = useState<fhir.DomainResource[]>([]);
```

Comment:
Why start the name with an underscore? I thought that traditionally meant "unused" but I see it's used below (line 37 as of now)

Reply:
Not sure if I wrote this specific change but I have been using this syntax to wrap the `setState` call in another method. Whenever `setPatientRecords` is run we also call `setEvaluatePath`. So the set method in the create here is `_setPatientRecords` and then on line 35 the actual `setPatientRecords` method is created:
```
const setPatientRecords = useCallback((value: fhir.DomainResource[]): void => {
    _setPatientRecords(value);
    setEvaluatePath(true);
  }, []);
```

Reply:
Oh I missed the fact that where it was being called was called `setPatientRecords`. It's a tricky naming convention if you aren't familiar with it.

Reply:
Do you have any suggestion? I saw it elsewhere in the code base and decided to mimick it in a few places